src/module/skb/event.rs

```diff
@@ -0,0 +1,91 @@
+use anyhow::{bail, Result};
+use serde::{Deserialize, Serialize};
+
+use super::bpf::*;
+use crate::core::events::{bpf::BpfRawSection, *};
+use crate::{EventSection, EventSectionFactory};
+
+/// Skb event section
+/// TODO: unflatten?
+#[derive(Default, Deserialize, Serialize, EventSection, EventSectionFactory)]
+pub(crate) struct SkbEvent {
+    // L2 fields
+    /// Ethertype.
+    pub(crate) etype: Option<u16>,
+    /// Source MAC address.
+    pub(crate) src: Option<String>,
+    /// Destination MAC address.
+    pub(crate) dst: Option<String>,
+    // IPv4 & IPv6 fields
+    /// Source IP address.
+    pub(crate) saddr: Option<String>,
+    /// Destination IP address.
+    pub(crate) daddr: Option<String>,
```

Comment:
Looking at events generated by this, it seems we need to add `#[serde(skip_serializing_if = "Option::is_none")]` or else the json event has a lot of `"whatever": null`

Reply:
Another idea to catch these ones (not necessarily to be added to this PR) is to have some unit tests that test event <-> json conversion.

Reply:
> > One thing to note is with the current implementation None values are converted to null in JSON (e.g. "netns":null). So all None values are present in the output. However I don't think we'll use JSON for stdout so we should have more control, and if needed we can use https://docs.rs/serde_with/latest/serde_with/attr.skip_serializing_none.html
> 
> With this we only should have to define a single attribute for the whole section. However I'm wondering if we need this right now, as I don't know if we'll use json for stdout. If json is only used to store data then I guess it's acceptable to have null values. WDYT?

I think it'll increase the size of the file unnecessarily.